irmin-demo.opam

```diff
@@ -0,0 +1,34 @@
+opam-version: "2.8"
+
+synopsis: "A mini-github applicatio"
+description: "This application is a sample project on how to combine irmin-graphql with dream and display queries on he browser using js_of_ocaml"
+
+license: "MIT"
+homepage: "https://github.com/dinakajoy/simple_mini_github"
+bug-reports: "https://github.com/dinakajoy/simple_mini_github/issues"
+
+author: "Odinaka Joy <dinakajoy@gmail.com> and Patrick Ferris <pf341@patricoferris.com>"
+
+depends: [
+  "dream"
+  "dune" {>= "2.8.0"} 
+  "js_of_ocaml-lwt" {= "3.11.0"}
+  "js_of_ocaml-ppx"
+  "ezjsonm" {>= "1.3.0"}
+  "brr" {>= "0.0.2"}
+  "note"
+  "str"
+  "astring"
+  "cohttp"
+  "fmt"
+  "git"
+  "irmin" {>= "2.9.1"}
```

Comment:
I think you will need an upper-bound here so you don't get `3+` which probably won't be compatible.